src/Funcionario.java

```diff
@@ -24,11 +24,39 @@ public String toString(){
         return "Nome: " + nome + "/" + "Salário: " + salario +
                 "/" + "Cargo: " + cargo + "Tempo de cargo: " + tempoDeCargo;
     }
-
+  
+    @Override
     public void promover() {
```

Comment:
que acha de trocar a resposta do método de void para Cargo?
Se o processo for bem sucedido, damos um ``return this.cargo`` , que é o novo cargo do funcionário
Se for mal sucedido, retornamos null, assim quem chamou o método saberá que ocorreu falha.

Reply:
Adiado para um próximo PR